.env

```diff
@@ -0,0 +1,22 @@
+INFLUXDB_VERSION=1.3.2-alpine
+INFLUXDB_INIT_VERSION=latest
+MYSQL_VERSION=5.5
+MYSQL_INIT_VERSION=1.5.1
+MEMCACHED_VERSION=1.5.0-alpine
+CADVISOR_VERSION=v0.26.1
+ZOOKEEPER_VERSION=3.4
+
+MON_SIDECAR_VERSION=1.0.0
+MON_KEYSTONE_VERSION=1.1.1
+MON_KAFKA_VERSION=0.9.0.1-2.11-1.1.0
+MON_KAFKA_INIT_VERSION=0.0.1
+MON_ALARMS_VERSION=1.1.0
+MON_GRAFANA_VERSION=4.0.0-1.1.1
+MON_GRAFANA_INIT_VERSION=1.1.0
+
+MON_API_VERSION=master
```

Comment:
It looks like we have some uniquely tagged (semver-ish) images and some that are just master or latest. With this users could end up with an invalid environment if they update from git and don't update any of their `master`-tagged images.

I think there's two options here:
 1. always use unique tags for everything (`master-{date}-{time}` instead of just `master`)
    - I tend to prefer this, and we need the unique tags for tagging stable releases (e.g. ocata) and for Helm anyway
    - The pr bot can make sure the master branch's `.env` always refers to the latest version of everything
      - side effect of this is that we periodically re-run CI, for better or for worse
      - needs to be manually maintained for a bit though
 2. document that users should run `docker-compose pull` immediately after running a `git pull`
    - requires few to no updates to either `docker-compose.yml` and `.env` into the future
    - could use `latest` for all semver images too
    - breaks the ability to check out some arbitrary git revision - can only guarantee a working environment from master or a stable OpenStack release
    - rollbacks between git revs are non-trivial